From 289c7c64c3e1b836fedd57b3e9f1eaa284e57d5f Mon Sep 17 00:00:00 2001
From: denglei <denglei@eswincomputing.com>
Date: Thu, 12 Dec 2024 15:40:07 +0800
Subject: [PATCH 337/652] WIN2030-16445:fix:fix the issue of no pcm node on
 die1.

Changelogs:
	fix the issue of no pcm node on die1.

Change-Id: I8941c25535883c22cb14e29eb85505843869d5cd
Signed-off-by: denglei <denglei@eswincomputing.com>
---
 .../boot/dts/eswin/eswin-win2030-die1-soc.dtsi    | 15 +++++++++------
 1 file changed, 9 insertions(+), 6 deletions(-)

diff --git a/arch/riscv/boot/dts/eswin/eswin-win2030-die1-soc.dtsi b/arch/riscv/boot/dts/eswin/eswin-win2030-die1-soc.dtsi
index 51734f0ecfe14..d7935bb5ec3bb 100644
--- a/arch/riscv/boot/dts/eswin/eswin-win2030-die1-soc.dtsi
+++ b/arch/riscv/boot/dts/eswin/eswin-win2030-die1-soc.dtsi
@@ -1197,8 +1197,9 @@ d1_pwm0: pwm@0x70818000 {
 
 		d1_i2s0: i2s0@70200000 {
 			compatible = "snps,i2s";
-			clocks = <&d1_clock WIN2030_CLK_VO_I2S_MCLK>;
-			clock-names = "mclk";
+			clocks = <&d1_clock WIN2030_CLK_VO_I2S_MCLK>,
+					<&d1_clock WIN2030_APLL_FOUT1>;
+			clock-names = "mclk", "apll";
 			#address-cells = <1>;
 			#size-cells = <0>;
 			#sound-dai-cells = <0x00000000>;
@@ -1215,8 +1216,9 @@ d1_i2s0: i2s0@70200000 {
 
 		d1_i2s1: i2s1@70210000 {
 			compatible = "snps,i2s";
-			clocks = <&d1_clock WIN2030_CLK_VO_I2S_MCLK>;
-			clock-names = "mclk";
+			clocks = <&d1_clock WIN2030_CLK_VO_I2S_MCLK>,
+					<&d1_clock WIN2030_APLL_FOUT1>;
+			clock-names = "mclk", "apll";
 			#address-cells = <1>;
 			#size-cells = <0>;
 			#sound-dai-cells = <0x00000000>;
@@ -1233,8 +1235,9 @@ d1_i2s1: i2s1@70210000 {
 
 		d1_i2s2: i2s@70220000 {
 			compatible = "snps,i2s";
-			clocks = <&d1_clock WIN2030_CLK_VO_I2S_MCLK>;
-			clock-names = "mclk";
+			clocks = <&d1_clock WIN2030_CLK_VO_I2S_MCLK>,
+					<&d1_clock WIN2030_APLL_FOUT1>;
+			clock-names = "mclk", "apll";
 			#address-cells = <1>;
 			#size-cells = <0>;
 			#sound-dai-cells = <0x00000000>;
-- 
2.51.1

